src/components/SiteLayout.tsx

```diff
@@ -85,6 +91,13 @@ const SiteLayout: FC<Props> = ({ children }) => {
     const currentUserAddress = currentUser?.ownedBy
     setMounted(true)
 
+    // Set mixpanel user id
+    if (currentUser?.id) {
+      Mixpanel.identify(currentUser?.id)
+    } else {
+      Mixpanel.identify('0x00')
+    }
```

Comment:
```suggestion
   Mixpanel.identify(currentUser?.id || '0x00')
```
may be? BTW don't chain same path twice

Reply:
We will set more user data if logged in!